package.json

```diff
@@ -19,7 +19,7 @@
     "./package.json": "./package.json",
     ".": {
       "types": "./dist/src/index.d.ts",
-      "module": "./dist/index.modern.js",
+      "module": "./dist/index.modern.mjs",
```

Comment:
It was intentional to have `.js`, otherwise it will break in some old bundlers.
Now, time passed. I think if this causes a problem, we should remove it entirely.
```suggestion
```
Does that fix?

Reply:
@dai-shi yes, it does fix.

Or could we add `"type": "module"` to `package.json`? this can also fix this issue

Reply:
> It was intentional to have `.js`, otherwise it will break in some old bundlers. Now, time passed. I think if this causes a problem, we should remove it entirely.


does that old bundler consume `exports` field? what I changed is only the one in `exports` field, the `module` field is kept as it is now

Reply:
`"type": "module"` might also break old bundlers. We keep cjs default for all jotai packages.

Now, would you be willing to fix more jotaijs/* packages? Some of them are not actively maintained. If the repo is migrated to pnpm from yarn, it means it's maintained.

Reply:
@dai-shi I find another solution, we can just reorder the keys of `exports["."]`, please review latest commit

Reply:
> Now, would you be willing to fix more jotaijs/* packages? Some of them are not actively maintained. If the repo is migrated to pnpm from yarn, it means it's maintained.

em... I'm afraid I donot have enough personal time to go throught other packages 😢

Reply:
The order was important for a bundler to pick "module" before "import".
So, instead of reordering, let's remove it.

Alright, let's wait for other contributors for other repos.

Reply:
👌 I've removed it

Reply:
Thanks.
Can you also remove "postcompile" script because we no longer need it?

Reply:
> Thanks. Can you also remove "postcompile" script because we no longer need it?

I think it's to be kept as `"module": "./dist/index.modern.js"` at Line 16 is not removed

Reply:
Oh, nice catch. Yeah, that might be good for very old bundlers (though, I'm not sure if we need to keep supporting it.)